extensions/netty/runtime/src/main/resources/META-INF/quarkus-extension.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Camel Quarkus MicroProfile Netty",
+  "labels": [
+    "integration",
+    "camel",
+    "netty"
+  ],
+  "guide": "https://quarkus.io/guides/camel"
+}
```

Comment:
We recently switched to yaml and there is a new required attribute `category` - see other extensions for examples.

Reply:
updated, copy from another, used 'categories' attribute, done